values-schema.yaml

```diff
@@ -1255,9 +1264,6 @@ properties:
         properties:
           _rawValues:
             $ref: '#/definitions/rawValues'
-          enabled:
```

Comment:
Looks like property removal. If so please add it to the values-changes.yaml file.

Reply:
@Morriz any comment here?

Reply:
same as before, it never existed in values as it was not bootstrapped.